src/qibolab/instruments/qblox/cluster_qrm_rf.py

```diff
@@ -988,10 +980,11 @@ def stop(self):
         try:
             self.device.stop_sequencer()
         except:
+            raise RuntimeError(f"Error stopping sequencer for {self.device.name}")
             pass
```

Comment:
```suggestion
            raise RuntimeError(f"Error stopping sequencer for {self.device.name}")
```

It's correct to raise an error, but then you don't need `pass` any longer.